Bottom: 327b4033a16c2fef165bad34b48ee66cd4e0f270
Top:    e1bc31f4bf8fbaca9a88034bbee52d28fc296845
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:42:43 -0500

Add new dependency, bc, to Dockerfile


---
diff --git a/docker/buildbot-worker-ptxdist/Dockerfile b/docker/buildbot-worker-ptxdist/Dockerfile
index ee6bfeb5..b14d49c4 100644
--- a/docker/buildbot-worker-ptxdist/Dockerfile
+++ b/docker/buildbot-worker-ptxdist/Dockerfile
@@ -22,6 +22,7 @@ RUN apt-get update \
 &&  DEBIAN_FRONTEND=noninteractive \
     apt-get -y install -q \
         autoconf \
+        bc \
         bison \
         build-essential \
         ccache \
